website/archiver/tasks.py

```diff
@@ -352,7 +352,17 @@ def archive_success(dst_pk, job_pk):
 
     # Update file references in the Registration's responses to point to the archived
     # file on the Registration instead of the "live" version on the backing project
-    utils.migrate_file_metadata(dst)
+    try:
+        utils.migrate_file_metadata(dst)
+    except ArchivedFileNotFound as err:
+        sentry.log_message(
+            f'Some files were not found while archiving the node {dst_pk}',
+            extra_data={
+                'missing_files': err.missing_files,
+            },
+        )
+        self.retry(exc=err)
```

Comment:
What's the purpose of the retry? Are you expecting a situation where the files will be found on the retry?